Library/SharedFunctionsTests.swift

```diff
@@ -404,4 +404,116 @@ final class SharedFunctionsTests: TestCase {
       checkoutPropertiesData.userHasStoredApplePayCard
     )
   }
+
+  func testGetShipping_ShippingEnabled() {
```

Comment:
This test appears to be actually testing a situation where you are testing the shipping amount based on an existing Backing so the name is not accurate imo. 

I would consider renaming this something like `testGetShipping_Backing` and writing another test for the situation when the Backing doesn't exist and the Reward has a non-zero shipping amount.